docs/quickstart.rst

```diff
@@ -136,21 +136,20 @@ but we're going to mostly avoid those here,
 just to show how to directly interface with the code's internal classes.
 
 First import some basic stuff from frank and load the data
-(again using the DSHARP observations of AS 209, available as a UVTable
+(again using the DSHARP observations of AS 209, available as a UVTable - with some reasonable time- and channel-averaging -
```

Comment:
Please add the details...

Reply:
I think the time averaging for the AS 209 UVTable is 10 s (or 30 s?) and the channel averaging 1 channel per spw (spectral window). @mtazzari is that correct? 

(I just wanted to add this detail in case people are comparing directly to the MS Table on the DSHARP website, but don't have to.)

Reply:
...I found the averaging used (stated in the frank ms). Updated in b49c8b1